index.js

```diff
@@ -1,6 +1,8 @@
 //https://github.com/angular/angular.js/pull/10732
 
-var angular = require('angular');
+if (typeof angular !== 'undefined') {
```

Comment:
I am referring to Variable hoisting
"Variable declarations, wherever they occur, are processed before any code is executed. The scope of a variable declared with var is its current execution context, which is either the enclosing function or, for variables declared outside any function"
[ https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/var](MDN Reference)

Please let me know if you think otherwise